tests/test_all.py

```diff
@@ -134,9 +134,8 @@ def test_cli_run_with_registered_models(
 
     for geojson_row in d["features"]:
         assert geojson_row["type"] == "Feature"
-        assert geojson_row["id"] == "PathTileObject"
+        assert geojson_row["id"] == "49069bbb-a0ac-4343-96fe-15af45c30634"
```

Comment:
after making the change to set id to a uuid, you'll have to update this test

```suggestion
        assert isinstance(geojson_row["id"], str)
        assert len(geojson_row["id"]) == 36
```